src/main/java/com/invicto/streaming_platform/web/controller/VideoController.java

```diff
@@ -50,4 +79,14 @@ public String incrementView(@RequestParam String id) {
         Path videoFile = fileService.findPathByVideoId(Long.parseLong(id));
         return ResponseEntity.ok().body(new FileSystemResource(videoFile));
     }
+
+    @PostMapping(value = "/video/add_comment")
+    public String addComment(@RequestParam Long id, @RequestParam String text) {
```

Comment:
Why use PostMapping if data are sent by RequestParam?
Use GetMapping with RequestParams or get data from RequestBody of Post Request.

Reply:
It's good that you noticed it. I'll replace it with GetMapping

Reply:
👌